Locales/enUS.lua

```diff
@@ -1,7 +1,7 @@
 local L = LibStub("AceLocale-3.0"):NewLocale("VocalRaidAssistant", "enUS", true)
 if not L then return end
 
-L["Credits"] = "Author: Nitrak, refactored by |cff0070daKyy|r of Blackrock"
+L["Credits"] = "Author: Nitrak, |cff0070daKyy|r of Blackrock"
```

Comment:
Let's make this a multi-line so it's not confusing that I'm not from Blackrock :P.

Aurthors:
Nitrak
|cff0070daKyy|r of Blackrock

Reply:
I wouldn't to this - one line looks cleaner

Reply:
It might, but then we need to set it up somehow differently. Right now it looks like we're both from Blackrock. You can give me Rogue colors and put "of Kazzak" instead and make the comma into an "and". That should work! :)